frontend/appflowy_flutter/lib/plugins/database/widgets/field/field_editor.dart

```diff
@@ -229,7 +237,7 @@ enum FieldAction {
   delete,
   wrap;
 
-  Widget? leading(FieldInfo fieldInfo, Color? color) {
+  Widget leading(FieldInfo fieldInfo, Color? color) {
```

Comment:
Please revert to returning `Widget?`. This adds unexpected, extra padding to the end of the row in the button. Otherwise, looks good to me.

Reply:
@richardshiue I don't think it will add padding, I have attached video and screenshot of the same, are you referring to this file https://github.com/Ahad-patel/AppFlowy/blob/e66835297a2dc84febf8d4f1517aa64252fc4624/frontend/appflowy_flutter/lib/workspace/presentation/home/menu/view/view_more_action_button.dart#L257, as explicit padding is added in it.

Or I am referring at wrong place, I don't know 🤷 ?.

https://github.com/user-attachments/assets/b13ac2df-9261-4f1a-aa8f-33c80b76a1dd


<img width="251" alt="Screenshot 2024-10-25 at 2 53 44 PM" src="https://github.com/user-attachments/assets/eaf97145-50d1-4642-8129-e7018c3643b2">
<img width="246" alt="Screenshot 2024-10-25 at 2 54 09 PM" src="https://github.com/user-attachments/assets/918f0122-d52a-43e7-a041-11075960020a">

Reply:
Yes that's it. Do you see that whereas edit property there is no extra padding at the end, the others have it? For languages where the string becomes too long, or we have another action that has a longer name, the right end will not line up

Reply:
I got your point. But since in FlowyIconTextButton , the leftIconBuilder and rightIconBuilder have a return type of Widget, so I have updated to Widget? and added a conditon that will check if resutant widget is nullable or not   `final Widget? leftIcon = leftIconBuilder?.call(onHover);` similar for the rightIconBuilder. So the extra padding will be avoided. Please let me know if any more changes needed for this PR. 

<img width="255" alt="Screenshot 2024-10-25 at 6 59 56 PM" src="https://github.com/user-attachments/assets/79dc16fa-e0cb-45e6-bd7f-fb824ad9749a">